project_from_quotation/readme/CONFIGURE.md

```diff
@@ -0,0 +1,7 @@
+Go to "Sales/Settings" and scroll to the "Quotations & Orders" section.
+
+Activate the "Create Projects From Quotation" checkbox and select a project to be used as a template in the "Template Project" field. 
+
+You can also select a pre-defined task template in the "Template Task" field. 
+
+NB: you can select from both active and archived projects/tasks in these fields.
```

Comment:
I have created a project and archived it. 

![image](https://github.com/OCA/sale-workflow/assets/2234999/19c0bf79-560f-4f92-a80f-7d311d7f9a64)


This project is not shown in the list:


![image](https://github.com/OCA/sale-workflow/assets/2234999/26c1be68-3a2a-4239-a52d-ff04397e6c29)